tests/e2e/classic_machine_pool_test.go

```diff
@@ -606,6 +606,54 @@ var _ = Describe("Create MachinePool", ci.Day2, ci.NonHCPCluster, ci.FeatureMach
 		Expect(mpResponseBody.InstanceType()).To(Equal("m5.2xlarge"))
 	})
 
+	It("can create machinepool with customized tags - [id:73942]", ci.High, ci.NonHCPCluster, func() {
+		By("Create machinepool creation plan with invalid tags")
```

Comment:
Why not having this step in case `63139` below instead ?

Reply:
Updated.